### PR TITLE
Rename and remove some files

### PR DESCRIPTION
**Pull request by mrocklin**
This renames some files and removes some old ones.  

@jrbourbeau I'm curious about your thoughts on this

@fjetter @phofl @hendrikmakait I'm doing this to try to streamline some of your experience at pycon-de.  I could be misguided though.

### CODE REVIEW

**Comment by mrocklin** [comment]
Ping @jrbourbeau (but no rush)

File: Dockerfile
Comment (jrbourbeau) [inline Dockerfile:1]: I'd like to keep this as it's used to generate our `coiled/gpu-examples` docker image that's used in the "Getting Started" section in the app.

**Review by jrbourbeau — APPROVED** [review]
Thanks @mrocklin. I'd like to keep the `Dockerfile` for now as it's used in the Coiled getting started guide. Otherwise this LGTM

**Comment by jrbourbeau** [comment]
Thanks @mrocklin

**Pull request merged**